src/modules/px4iofirmware/px4io.c

```diff
@@ -377,6 +377,10 @@ user_start(int argc, char *argv[])
 	if (minfo.mxordblk < 550) {
 
 		syslog(LOG_ERR, "ERR: not enough MEM");
+
+		/* schedule a reboot to give the FMU a chance to update */
```

Comment:
```suggestion
		/* schedule a reboot in 2 seconds to give the FMU a chance to update */
```

Reply:
My only questions is will this beat again the FMU boot up. Will the IO be in police lights while the FMU is trying to force program it.

As an alternative we could enable the comms and keep the police lights but reboot on a reboot command.